zeppelin-web/src/components/modal-shortcut/modal-shortcut.html

```diff
@@ -125,7 +125,6 @@ <h4 class="modal-title" id="myModalLabel">Keyboard shortcuts</h4>
         <div class="row">
           <div class="col-md-4">
             <div class="keys">
-              <kbd class="kbd-dark">Ctrl</kbd> + <kbd class="kbd-dark">{{ isMac ? 'Option' : 'Alt'}}</kbd> + <kbd class="kbd-dark">o</kbd>
               <kbd class="kbd-dark">Ctrl</kbd> + <kbd class="kbd-dark">Alt</kbd> + <kbd class="kbd-dark">r</kbd>
```

Comment:
it looks like this `Alt` should be `{{ isMac ? 'Option' : 'Alt'}}`? Basically everywhere we have Alt should say Option on Mac